src/documentation/0001-node-introduction/index.md

```diff
@@ -96,6 +96,7 @@ Node.js is a low-level platform. In order to make things easy and exciting for d
 Many of those established over time as popular options. Here is a non-comprehensive list of the ones worth learning:
 
 * [**AdonisJs**](https://adonisjs.com/): A full-stack framework highly focused on developer ergonomics, stability, and confidence. Adonis is one of the fastest Node.js web frameworks.
+* [**Egg.js**](https://eggjs.org/en/): A framework to build better enterprise frameworks and apps with Node.js & Koa.
```

Comment:
Is it really a framework to build frameworks? (If it is, then great. But if it isn't, let's fix that up!)

Reply:
I believe so. It exposes a capability to create a code re-usage across different projects like plugins but much more seamlessly, just like a new framework, documented here https://eggjs.org/en/advanced/framework.html .